### PR TITLE
Update quick-start-with-tidb.md

### PR DESCRIPTION
**Pull request by lucklove**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glkappe** [comment]
66

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -67,7 +67,12 @@ title: TiDB 数据库快速上手指南
         To view the dashboard: http://127.0.0.1:2379/dashboard
         To view the monitor: http://127.0.0.1:9090
         ```
-
+        
+        > **注意：**
+        >
+        > 以这种方式执行的 playground，在运行结束后 TiUP 会清理掉其数据，重新执行该命令后会得到一个全新的集群。
+        > 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup --tag <your-tag> playground ...`，详情请参考 [TiUP 参考手册](https://docs.pingcap.com/zh/tidb/stable/tiup-reference#-t---tag-string)。
```
Comment (yikeke) [inline quick-start-with-tidb.md:74]:
```suggestion
        > 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup --tag <your-tag> playground ...`，详情参考 [TiUP 参考手册](/tiup/tiup-reference.md#-t---tag-string)。
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -67,7 +67,12 @@ title: TiDB 数据库快速上手指南
         To view the dashboard: http://127.0.0.1:2379/dashboard
         To view the monitor: http://127.0.0.1:9090
         ```
-
+        
+        > **注意：**
+        >
+        > 以这种方式执行的 playground，在运行结束后 TiUP 会清理掉其数据，重新执行该命令后会得到一个全新的集群。
```
Comment (yikeke) [inline quick-start-with-tidb.md:73]:
```suggestion
        > 以这种方式执行的 playground，在运行结束后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。
```

**Pull request merged**